src/pudl/transform/eia861.py

```diff
@@ -194,79 +622,779 @@ def service_territory(raw_dfs, tfr_dfs):
+    )
+
+    # Organize col headers for output
+    transformed_gp = pudl.helpers.organize_cols(
+        transformed_gp, idx_cols + ['utility_name_eia', 'customer_class'])
+
+    tfr_dfs["green_pricing_eia861"] = transformed_gp
+
+    return tfr_dfs
+
+
+def mergers(tfr_dfs):
     """
-    Transforms EIA 860 DataFrames.
+    Transform the EIA 861 Mergers table.
+
+    Args:
+        tfr_dfs (dict): A dictionary of transformed EIA 861 DataFrames, keyed by table
+            name. It will be mutated by this function.
+
+    Returns:
+        dict: A dictionary of transformed EIA 861 dataframes, keyed by table name.
+
+    """
+    raw_mergers = tfr_dfs["mergers_eia861"].copy()
+
+    # No data tidying required
+
+    ###########################################################################
+    # Transform Values:
+    # * Turn ownership column from single-letter code to full ownership category.
+    # * Retain preceeding zeros in zip codes
+    ###########################################################################
+
+    transformed_mergers = (
+        raw_mergers.assign(
+            entity_type=lambda x: x.entity_type.map(pc.ENTITY_TYPE_DICT),
+            merge_zip_5=lambda x: pudl.helpers.zero_pad_zips(x.merge_zip_5, 5),
+            merge_zip_4=lambda x: pudl.helpers.zero_pad_zips(x.merge_zip_4, 4)
+        )
+    )
+
+    # No duplicates to speak of but take measures to check just in case
+    _check_for_dupes(transformed_mergers, 'Mergers', [
+                     "utility_id_eia", "state", "report_date"])
+
+    # Organize col headers for output
+    transformed_mergers = (
+        pudl.helpers.organize_cols(
+            transformed_mergers, ['utility_id_eia', 'utility_name_eia',
+                                  'state', 'report_date']
+        )
+    )
+
+    tfr_dfs["mergers_eia861"] = transformed_mergers
+    return tfr_dfs
+
+
+def net_metering(tfr_dfs):
+    """
+    Transform the EIA 861 Net Metering table.
+
+    Args:
+        tfr_dfs (dict): A dictionary of transformed EIA 861 DataFrames, keyed by table
+            name. It will be mutated by this function.
+
+    Returns:
+        dict: A dictionary of transformed EIA 861 dataframes, keyed by table name.
+
+    """
+    idx_cols = [
+        'utility_id_eia',
+        'state',
+        'report_date',
+        'balancing_authority_code_eia'
+    ]
+
+    # Pre-tidy clean specific to sales table
+    raw_nm = tfr_dfs["net_metering_eia861"].copy()
+
+    ###########################################################################
+    # Tidy Data:
+    ###########################################################################
+
+    logger.info("Tidying the EIA 861 Net Metering table.")
+    # Normalize by customer class
+    tidy_nm = _tidy_class_dfs(raw_nm, 'Net Metering',
+                              idx_cols, CUSTOMER_CLASSES, 'customer_class')
+    # maybe delete these cols from excel so there aren't duplicates?
+    tidy_nm = tidy_nm.drop(columns='customers')
```

Comment:
What does the `customers` column refer to? Is it the number of customers with certain net metering tech? It is not clear to me why this is being dropped here.

Also to address the comment about deleting things from excel, we generally want to steer clear of modifying the original files. Mostly because any change made is challenging to track and ensure everyone running pudl has the same changes. Also we like to "refresh" our original data files whenever the sources come out with new versions.

Reply:
That makes sense RE: deleting things from excel. The reason I've dropped the customers column, is because it's repetitive. The customer columns being dropped are `residential_customers`, `commercial_customers`, etc. Each fuel class, however, is broken into customers as well. There exists, for instance: `residential_pv_customers`, `residential_wind_customers` -- a more granular version of the former. I can run a triple check to see that they are the same values.

Reply:
Another thing worth discussing is the name `fuel_classes` I've set up for this transform module. I chose it because it encompasses the different fuel types reflected in this `net_metering` table in particular. I wanted to double check and see whether you wanted to try and merge this with another fuel classification column or keep it separate. `Fuel_class` makes sense as a name in this module (as a comparison to `customer_class` but it's not comprehensive when compared to 860 or other tables that might have a fuel-related prefix.

Reply:
ahhh okay. this makes sense! Thanks for the clarification. Maybe pull this explanation into an in-line comment so the reasoning will be preserved.